# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome14/gnome-libs14/compile.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- gnome-libs-1.4.1.7/gnome-config.in	2001-03-05 23:17:43.000000000 +0100
+++ new/gnome-config.in	2002-05-26 00:24:18.000000000 +0200
@@ -19,7 +19,7 @@
 pkglibdir=${libdir}/@PACKAGE@
 pkgincludedir=${includedir}/@PACKAGE@
 MODULE_VERSION=@PACKAGE@-@VERSION@
-module_dirs="$libdir"
+module_dirs="$libdir /usr/lib"
 
 # stolen from autoconf
 if (echo "testing\c"; echo 1,2,3) | grep c >/dev/null; then
@@ -113,6 +113,9 @@
 modversion=false
 moddatadir=false
 
+the_libs=""
+the_flags="-I${includedir}"
+
 while test $# -gt 0; do
     case "$1" in
     -*=*) optarg=`echo "$1" | sed 's/[-_a-zA-Z0-9]*=//'` ;;
